src/openrct2/drawing/NewDrawing.cpp

```diff
@@ -23,12 +23,9 @@
 #include "../paint/Painter.h"
 #include "IDrawingContext.h"
 #include "IDrawingEngine.h"
-#include "NewDrawing.h"
```

Comment:
Are you sure about this?

Reply:
No. I'm going to close this PR as it's impossible to thoroughly check all includes manually. Sorry :/ But I learned something :)